apps/storybook/src/VisCanvasInteraction.stories.tsx

```diff
@@ -9,11 +9,11 @@ interface TemplateProps {
   panZoom?: boolean;
   tooltipValue?: string;
   guides?: TooltipMeshProps['guides'];
-  drawSelection?: boolean;
+  selection: 'line' | 'rectangle' | 'none';
```

Comment:
I wanted to have a radio button as control to be able to disable `selection`.

The issue with `?`is that you can't set the arg back to `disabled` once you have clicked on the radio button. This is the case for `guides` for example.

Or am I missing something ?

Reply:
Ah right, I see. What about a drop-down menu? Otherwise, maybe put `none` as the default value? It's a bit strange that `none` is equivalent to all the radios being deselected.

Reply:
Well, in the end, making two stories (one for the line and one for the rectangle) made the arg superfluous in my opinion. So I removed it completely.